spec/support/authentication_helpers.rb

```diff
@@ -1,25 +1,27 @@
 # frozen_string_literal: true
 
 module AuthenticationHelpers
-  mattr_accessor :username, :password
+  mattr_accessor :username, :password, :token
 
   def authenticate!(start_url:, expected_text:)
     # View the specified starting URL
     visit start_url
 
+    return if page.has_text?(expected_text, wait: Settings.post_authentication_text_timeout)
```

Comment:
This is a check we used to make back in the old "automatic authentication" days (read: a couple hours ago) that only trips you up if 1. you're using Firefox which authenticates only once even when running multiple tests, and 2. running multiple tests.

Reply:
This would be good for @justinlittman and @ndushay (who helped earlier) to test on their machines. I'd recommend:

```shell
bundle exec rspec spec/features/apo_creation_spec.rb spec/features/sdr_deposit_spec.rb
```